README.md

```diff
@@ -9,11 +9,25 @@ An openvpn config file either on the openvpn server or in the inventory and an a
 
 ## Role Variables
 
-| Variable                          | Default / Mandatory        | Description                                                                                                                       |
-|-----------------------------------|----------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
-| `openvpn_client_server`           | :heavy_multiplication_x:   | Inventory name of the host running the [openvpn role](https://github.com/stuvusIT/openvpn)                                        |
-| `openvpn_client_name`             | `{{ inventory_hostname }}` | Client name used in on the openvpn server                                                                                         |
-| `openvpn_client_config_file_name` | `openvpn.ovpn`             | File name to copy over, only used when `openvpn_client_server` is not set. Looks for the file in `files/{{ inventory_hostname }}/ |
+| Variable                           | Default                    | Description                                                                             |
+| ---------------------------------- | -------------------------- | --------------------------------------------------------------------------------------- |
+| `openvpn_client_name`              | `{{ inventory_hostname }}` | Name of the client; doesn't need to match the host name                                 |
+| `openvpn_client_server`            | `default-server`           | Ansible host name of the OpenVPN server                                                 |
+| `openvpn_client_local_config_path` |                            | The OpenVPN client configuration to use, [read below](#client-configuration)            |
+| `openvpn_client_up_commands`       | `[]`                       | List of commands thats ran as soon as the OpenVPN TAP/TUN interface goes up             |
+| `openvpn_client_extra_config`      | `[]`                       | List of extra lines that are, if not already present, added to the client configuration |
+
+### Client configuration
+
+There are two ways to configure the OpenVPN client:
+
+* Specifying a configuration file in `openvpn_client_local_config_path`.
+* Pulling the configuration file from the OpenVPN server.
+  This is done when `openvpn_client_confi_filename` is **not** specified.
+  The OpenVPN server is then required to be an Ansible host with hostname `openvpn_client_server`.
+  It needs to have the configuration file, which is to be pulled, at `/etc/openvpn/{{ openvpn_client_name }}-{{ openvpn_client_server }}.ovpn`.
```

Comment:
This assumes `openvpn_base_dir` on the OpenVPN server is set to `/etc/openvpn`, just a nitpick though, I wouldn't change it. Also, the file has to be base64 encoded.

Reply:
Our openvpn (server) role doesn't work if `openvpn_base_dir` is changed. This is because the official OpenVPN systemd service always assumes that directory to be `/etc/openvpn`.